apps/server/src/controller/dex_token_controller.ts

```diff
@@ -1,4 +1,5 @@
 import { body, Context, request, responses, summary, tags, description } from 'koa-swagger-decorator';
+import { TokenTokenHolderFactory } from '../model';
```

Comment:
TokenHolderFactory?

Reply:
> TokenHolderFactory?
Singleton pattern, wrapping the process of reading the token json file

Reply:
> TokenHolderFactory?

This side is for testing and will be removed during business development